src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -198,10 +198,7 @@ describe('ReactDOMComponent', function() {
 
       expect(console.error.argsForCall.length).toBe(1);
       expect(console.error.argsForCall[0][0]).toEqual(
-        'Warning: `span` was passed a style object that has previously been ' +
-        'mutated. Mutating `style` is deprecated. Consider cloning it ' +
-        'beforehand. Check the `render` using <span>. Previous style: ' +
-        '{fontSize: NaN}. Mutated style: {fontSize: NaN}.'
+        'Warning: `NaN` is an invalid value for the `fontSize` css style property'
```

Comment:
I actually want to move the warnings out into a separate module eventually (using devtools api), so if we move it to CSSPropertyOperations, I would just have to move it back here later.  I think it's fine to leave it here.